test/e2e/storage/generic_persistent_volume-disruptive.go

```diff
@@ -102,7 +102,9 @@ func createPodPVCFromSC(f *framework.Framework, c clientset.Interface, ns string
 	framework.ExpectEqual(len(pvs), 1)
 
 	ginkgo.By("Creating a pod with dynamically provisioned volume")
-	pod, err := framework.CreateNginxPod(c, ns, nil, pvcClaims)
+	pod, err := framework.CreateSecPod(c, ns, pvcClaims, nil,
+		false, "", false, false, framework.SELinuxLabel,
+		nil, framework.PodStartTimeout)
```

Comment:
Why do we need to change here?
I cannot find the reason from the PR message.

Reply:
`framework.CreateNginxPod` can't handle block volume and inline volume, but `framework.CreateSecPod` can.
Current tests in generic_persistent_volume-disruptive.go doesn't actually test them, but it would be good to use the similar logic to testsuites/disruptive.go to prepare for future imporvement and maintenance.